scripts/start.sh

```diff
@@ -28,6 +28,11 @@ mkdir -p ${DIR}/security/keypair
 openssl genrsa -out ${DIR}/security/keypair/keypair.pem 2048
 openssl rsa -in ${DIR}/security/keypair/keypair.pem -outform PEM -pubout -out ${DIR}/security/keypair/public.pem
 
+# Do not make keys world-readable in production
```

Comment:
```suggestion
echo "SETTING INSECURE PERMISSIONS ON '${DIR}/security' FOR DEMO PURPOSES ONLY NEVER USE THIS IN PRODUCTION"
# Do not make keys world-readable in production
```

Reply:
@andrewegel good idea to echo this out.  I wordsmithed it and committed in latest commit.